config/sidekiq.yml

```diff
@@ -10,18 +10,19 @@
 
 :schedule:
   CalculateAllMetrics:
-    cron: '0 0 * * * *' # Run at midnight
+    cron: '0 0 0 * * *' # Run at midnight
     class: CalculateAllMetricsJob
     queue: scheduled
   PreloadOpenData:
-    cron: '0 1 * * * *' # Run at 01:00
+    cron: '0 0 1 * * *' # Run at 01:00
```

Comment:
Here comment are not representative of the current cron config 
'0 0 1 * * *' => "At 12:00 AM, on day 1 of the month
https://crontab.cronhub.io/

Reply:
Your generator doesn't give me the same answer ...
![image](https://user-images.githubusercontent.com/464350/145185960-9a37bb71-e4e3-4478-83d4-93e3fbc7323e.png)